.github/workflows/conda-shep.yml

```diff
@@ -0,0 +1,62 @@
+name: Build env scikit-hep
+
+on: [push, pull_request]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 5
+      matrix:
+        os: ["ubuntu-latest", "macos-latest"]
+        python-version: ["3.9"]
```

Comment:
Being maniac I would also test 3.8 as many users are bound to be on that major version.

Reply:
The environment specifies 3.9, and the users are expected to setup the environment, so how can they not have 3.9?

Reply:
I mean, for anyone installing locally, not using Binder, will you demand that they have 3.9? If that's the case then fine.

Reply:
I always try to be as flexible as possible, hence my comment.

Reply:
The users are asked to create the environment and that will install the specified python version.

Reply:
OK. I will try the setup next week anyway as on my Windows laptop I get 3.8 from Anaconda, which is what I use "daily".